src/data/valid/MassSpectrometry-NOM.yaml

```diff
@@ -0,0 +1,11 @@
+id: nmdc:dgms-99-oW43DzG1
+type: nmdc:MassSpectrometry
+has_input:
+  - nmdc:procsm-11-9gjxns61
+has_output:
+  - nmdc:dobj-00-9n9n9n
+has_configuration: nmdc:mscon-99-oW43DzG0
+eluent_introduction: nmdc:cspro-99-hello00
```

Comment:
do you know if we have any example data file that populate an eluant introduction instance? Is that still going to be relevant after you do some chromatography configuration modeling?

Reply:
Yes, this will still be relevant.  I will provide an extend these MassSpectrometry examples alongside the parallel work I've been doing for the `ChromatographicSeparationConfiguration` work.